conans/client/generators/compiler_args.py

```diff
@@ -64,6 +66,11 @@ def content(self):
         flags.extend(self._deps_build_info.sharedlinkflags)
         flags.extend(self._deps_build_info.exelinkflags)
         flags.extend(self._libcxx_flags())
+        if is_apple_os(the_os):
```

Comment:
I thought we agreed about not introducing the check. I think it is better to simplify.

Reply:
yeah, just a misunderstanding from my side, I'll revert back

Reply:
removed the condition